__tests__/app.test.js

```diff
@@ -150,6 +150,26 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+describe("/api/users get request", () => {
+    test("should respond with a staus of 200 and an array of 4 user objects", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body: { users } }) => {
+          expect(users).toHaveLength(4);
+        });
+    });
+    test("should respond with a status of 200 and an array of user objects each with a username property", () => {
```

Comment:
You could merge these tests. With an empty array, one test will fail and another will pass. If merged, an empty array cannot pass the test